indico/modules/events/management/forms.py

```diff
@@ -216,6 +216,8 @@ def validate_contact_emails(self, field):
 
 
 class EventClassificationForm(IndicoForm):
+    _fields_order = ('keywords', 'references', 'label', 'label_message')
```

Comment:
It's called `_field_order` in other places so let's stick with that :)
```suggestion
    _field_order = ('keywords', 'references', 'label', 'label_message')
```

Reply:
Fair, fixed!